jobs/config.json

```diff
@@ -8074,6 +8074,28 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-kops-aws-kopeiovxlan": {
+    "args": [
+      "kubetest",
+      "--",
+      "--deployment=kops",
+      "--extract=ci/latest",
+      "--kops-kubernetes-version=ci/latest",
+      "--up",
+      "--test",
+      "--down",
+      "--kops-cluster=e2e-kops-aws-kopeiovxlan.test-aws.k8s.io",
+      "--kops-args=--networking=kopeio-vxlan",
+      "--kops-state=s3://k8s-kops-jenkins/",
+      "--test_args=--ginkgo.focus=ConfigMap --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]",
+      "--v"
+    ],
+    "scenario": "execute",
```

Comment:
Good point.  I would say that I didn't consider these logs, rather than I don't think it matters :-)

I am starting with a new e2e test, rather than risking one of the existing ones...